### PR TITLE
Upgrade primitives

### PR DESCRIPTION
**Pull request by peduarte**
Upgrading the Primitives

### CODE REVIEW

**Review by benoitgrelard — COMMENTED** [review]
There are some breaking changes to handle right?
Tooltip, dialogs, etc?

**Comment by peduarte** [comment]
@benoitgrelard oh, i misunderstood the change log 🤔 technically nothing's broken now, looks like we need to use the new `Portal` part if we want to keep them portalled. interesting, the releases don't include any guidance on this. has this been documented anywhere?

also, we should probably consider adding: "Supports modal and non-modal modes." in the doc features

**Review by colmtuite — APPROVED** [review]

**Comment by benoitgrelard** [comment]
@peduarte there are a few new versions for small fixes we shipped today that are part of the public release.
I think it's `Tooltip`, `Dialog` and `AlertDialog`, just version changes, no other change required.

File: components/DesignSystemProvider.tsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import React from 'react';
-import { IdProvider } from '@radix-ui/react-id';
+import { TooltipProvider } from '@radix-ui/react-tooltip';
 
-export const DesignSystemProvider: React.FC = ({ children }) => <IdProvider>{children}</IdProvider>;
+export const DesignSystemProvider: React.FC = ({ children }) => (
+  <TooltipProvider>{children}</TooltipProvider>
+);
```
Comment (jjenzz) [inline components/DesignSystemProvider.tsx:6]: If we do this, everyone who uses the DS would have the `Tooltip` in their bundle even if they're not using it. Is that something we're concerned about?

Comment (colmtuite) [reply components/DesignSystemProvider.tsx:6]: It shouldn't be an issue for us at Modulz, all our apps use almost all of the components.

Comment (peduarte) [reply components/DesignSystemProvider.tsx:6]: Yeah, this is mainly for us

**Comment by peduarte** [comment]
@benoitgrelard good shout, upgraded

**Comment by benoitgrelard** [comment]
Sorry @peduarte there's another round of udpates

**Pull request closed without merging**